Bug: tests in `tests/unit/test_channels/test_subscriber.py` fail for unrelated PR

### Description

This happened on 2 different test reruns:

```
FAILED tests/unit/test_channels/test_subscriber.py::test_subscriber_backlog_backoff - RuntimeError: There is no current event loop in thread 'MainThread'.
FAILED tests/unit/test_channels/test_subscriber.py::test_subscriber_backlog_dropleft - RuntimeError: There is no current event loop in thread 'MainThread'.
= 2 failed, 5558 passed, 8 skipped, 13 xfailed, 30 xpassed, 4 warnings in 93.83s (0:01:33) =
```

Link: https://github.com/litestar-org/litestar/actions/runs/12687347653/job/35361920794?pr=3935

The PR is just a typo fix: https://github.com/litestar-org/litestar/pull/3935

### URL to code causing the issue

_No response_

### MCVE

_No response_

### Steps to reproduce

_No response_

### Screenshots

_No response_

### Logs

_No response_

### Litestar Version

`main`

### Platform

- [X] Linux
- [ ] Mac
- [ ] Windows
- [ ] Other (Please specify in the description above)

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -193,6 +193,11 @@ nitpick_ignore = [
     (PY_CLASS, "advanced_alchemy.types.BigIntIdentity"),
     (PY_CLASS, "advanced_alchemy.types.JsonB"),
     (PY_CLASS, "advanced_alchemy.repository.SQLAlchemyAsyncRepository"),
+    # docs in flux as we prepare for `advanced_alchemy` 1.0 release. re-enable when finished
+    (PY_CLASS, "advanced_alchemy.base.UUIDBase"),
+    (PY_CLASS, "advanced_alchemy.base.UUIDAuditBase"),
+    (PY_CLASS, "advanced_alchemy.base.BigIntBase"),
+    (PY_CLASS, "advanced_alchemy.base.BigIntAuditBase"),
 ]
 
 nitpick_ignore_regex = [

--- a/docs/examples/testing/test_subprocess_sse.py
+++ b/docs/examples/testing/test_subprocess_sse.py
@@ -16,18 +16,11 @@ from litestar.testing import subprocess_async_client
 if sys.platform == "win32":
     asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
 
-pytestmark = pytest.mark.anyio
-
-
-@pytest.fixture(scope="session")
-def anyio_backend() -> str:
-    return "asyncio"
-
 
 ROOT = pathlib.Path(__file__).parent
 
 
-@pytest.fixture(name="async_client", scope="session")
+@pytest.fixture(name="async_client")
 async def fx_async_client() -> AsyncIterator[httpx.AsyncClient]:
     async with subprocess_async_client(workdir=ROOT, app="subprocess_sse_app:app") as client:
         yield client

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -213,6 +213,7 @@ fail_under = 50
 [tool.pytest.ini_options]
 addopts = "--strict-markers --strict-config --dist=loadgroup -m 'not server_integration'"
 asyncio_mode = "auto"
+asyncio_default_fixture_loop_scope = "function"
 filterwarnings = [
   "error",
   # https://github.com/pytest-dev/pytest-asyncio/issues/724

--- a/tests/unit/test_channels/test_subscriber.py
+++ b/tests/unit/test_channels/test_subscriber.py
@@ -8,11 +8,10 @@ import pytest
 from litestar.channels import Subscriber
 from litestar.channels.subscriber import BacklogStrategy
 from litestar.utils.compat import async_next
+from tests.unit.test_channels.util import get_from_stream
 
-from .util import get_from_stream
 
-
-def test_subscriber_backlog_backoff() -> None:
+async def test_subscriber_backlog_backoff() -> None:
     subscriber = Subscriber(plugin=MagicMock(), max_backlog=2, backlog_strategy="backoff")
 
     assert subscriber.put_nowait(b"foo")
@@ -23,7 +22,7 @@ def test_subscriber_backlog_backoff() -> None:
     assert [subscriber._queue.get_nowait(), subscriber._queue.get_nowait()] == [b"foo", b"bar"]
 
 
-def test_subscriber_backlog_dropleft() -> None:
+async def test_subscriber_backlog_dropleft() -> None:
     subscriber = Subscriber(plugin=MagicMock(), max_backlog=2, backlog_strategy="dropleft")
 
     assert subscriber.put_nowait(b"foo")

--- a/tests/unit/test_testing/test_sub_client/test_subprocess_client.py
+++ b/tests/unit/test_testing/test_sub_client/test_subprocess_client.py
@@ -16,17 +16,16 @@ from litestar.testing import subprocess_async_client, subprocess_sync_client
 if sys.platform == "win32":
     asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
 
-
 ROOT = pathlib.Path(__file__).parent
 
 
-@pytest.fixture(name="async_client", scope="session")
+@pytest.fixture(name="async_client")
 async def fx_async_client() -> AsyncIterator[httpx.AsyncClient]:
     async with subprocess_async_client(workdir=ROOT, app="demo:app") as client:
         yield client
 
 
-@pytest.fixture(name="sync_client", scope="session")
+@pytest.fixture(name="sync_client")
 def fx_sync_client() -> Iterator[httpx.Client]:
     with subprocess_sync_client(workdir=ROOT, app="demo:app") as client:
         yield client

